feat(tqdm): #39 Implemented progress bar for orchestrator:main, clien…

cesarnunezh opened the pull request:
Implemented progress bar for orchestrator:main, client:main, DocumentService:batch_download, and tests

cesarnunezh committed:
feat(tqdm): #39 Implemented progress bar for orchestrator:main, client:main, DocumentService:batch_download

ymohanty commented:
@cesarnunezh  Some tests fail. I think it's a `ModuleImportError`. Perhaps you didn't update the uv lockfile?

cesarnunezh committed:
add colorama

ymohanty committed:
Merge branch 'dev' into tqdm

ymohanty committed:
chore(client): Remove redundant `track_with_progress` function. TODO: move this function to a utilities file

ymohanty committed:
chore(orchestrator): Use standard `stop_logging_to_console` and `resume_logging_to_console` functions to handle logging

Route the logs to log files instead of the console to reduce clutter. Show tqdm progress bars on console

ymohanty committed:
fix(orchestrator): Remove circular references to track with progress
